rfcs/0003-rfc-process/README.md

```diff
@@ -0,0 +1,119 @@
+---
+author: Roni Choudhury
+date: 2020-06-29
+---
+
+# RFC: Adopt Django-esque process for creating RFCs
+
+This RFC describes a process for creating RFCs that adapts Django's methodology
+for creating "DEPs" (Django Enhancement Proposals). [Their
+process](https://github.com/django/deps/blob/master/final/0001-dep-process.rst)
+is detailed via its own DEP.
+
+## Motivation
+
+It turns out that our RFC documents have multiple paths towards resolution: some
+of them are accepted as intended, and become useful information, or actionable
+plans. Others are modified in flight and settle in a different form than
+intended, while others are rejected. In order to track these paths, as well as
+provide a distinction between "informational" and "actionable", we intend to
+adopt a slightly richer process adapted from Django's use of "DEPs" (which in
+turn is adapted from Python's famous "PEPs").
+
+The main additions to our process are:
+- a directory structure between which proposal move as their status changes
+- a template that is to be copied into new draft RFCs to avoid the difficulty of
+  getting started
+- different resolved states, including `accepted` for proposals that are accepted;
+  `final` for ones that have been accepted *and* implemented; `superseded` for
+  ones that have been made obsolote by newer RFCs; `rejected` for those are
+  rejected (but kept for informational/historical value); `withdrawn` for those
+  the author wishes to abandon; and `draft` as a place for in-flight RFCs can be
+  kept
+
+## Proposals
+
+### New Process for RFC Creation/Review/Acceptance/Rejection
+
+1. The author clones this repository, creates a new directory in the `drafts`
+   directory with `X`s in place of the RFC number, (e.g.,
+   `drafts/XXXX-new_proposal/`), and copies the root-level `template.md` file
+   there as `README.md`.
```

Comment:
If _this_ RFC is accepted, would this `template.md` be supplied after this is in the `accepted` stage?

Reply:
Yes. I just didn't want to go to the trouble before at least preliminary buy-in.